pkg/kubectl/metricsutil/metrics_printer.go

```diff
@@ -172,6 +197,31 @@ func printSinglePodMetrics(out io.Writer, m *metricsapi.PodMetrics, printContain
 	return nil
 }
 
+func getPodMetrics(m *metricsapi.PodMetrics, printContainersOnly bool) (v1.ResourceList, map[string]v1.ResourceList, error) {
+	containers := make(map[string]v1.ResourceList)
+	podMetrics := make(v1.ResourceList)
+	for _, res := range MeasuredResources {
+		podMetrics[res], _ = resource.ParseQuantity("0")
+	}
+
+	for _, c := range m.Containers {
+		var usage v1.ResourceList
+		err := scheme.Scheme.Convert(&c.Usage, &usage, nil)
```

Comment:
can be:
`if err := ......; err != nil {`

Reply:
Ditto wrt conversions being expensive.